lua/nvim-treesitter/highlight.lua

```diff
@@ -175,104 +41,4 @@ function M.detach(bufnr)
   enable_syntax(bufnr)
 end
 
--- TODO(clason): remove obsolete function after bump to 0.8
-function M.set_custom_captures(captures)
-  for capture, hlgroup in pairs(captures) do
```

Comment:
Removing this probably requires updating the readme

Reply:
Ah, missed that. I'll make a fixup PR.